src/rules/converters/ban-types.ts

```diff
@@ -1,10 +1,37 @@
 import { RuleConverter } from "../converter";
 
-export const convertBanTypes: RuleConverter = () => {
+export const convertBanTypes: RuleConverter = tslintRule => {
+    type ConvertBanTypeArgument = {
+        message: string;
+    };
+
+    const bannedTypesObj: { [key: string]: ConvertBanTypeArgument | null } = {};
+
+    for (const rule of tslintRule.ruleArguments) {
+        if (!Array.isArray(rule)) {
+            break;
+        }
+
+        const typ = rule[0];
```

Comment:
Nit: if you want to be really slick ✨ 😉 you could do something like:

```ts
const [bannedType, message] = rule;
```

...so you can turn the object assigned to `bannedTypesObj[typ]` into `{ message }`.